app/src/main/java/com/magdamiu/androidfundamentalsmai2021/MainActivity.java

```diff
@@ -110,11 +109,8 @@ private void displayLogs() {
     }
 
     private void clickOnDisplayGreetingsButton() {
-        buttonDisplayGreetings.setOnClickListener(new View.OnClickListener() {
-            @Override
-            public void onClick(View view) {
+        buttonDisplayGreetings.setOnClickListener(view -> {
```

Comment:
Please clean up the empty lines before sending the code for review. A good reference about this topic plus many more is the book Clean Code